cMake/FindXercesC.cmake

```diff
@@ -30,6 +30,18 @@ FIND_LIBRARY(XercesC_LIBRARIES
     /usr/${LIB_DESTINATION}
 )
 
+FIND_LIBRARY(XercesC_LIBRARIE_DEBUG
+  NAMES
+    xerces-c_3d
+  PATHS
+    ${CMAKE_LIBRARY_PATH}
+    "[HKEY_CURRENT_USER\\software\\xerces-c\\lib]"
+    "[HKEY_CURRENT_USER\\xerces-c\\lib]"
+    $ENV{XERCESCROOT}/${LIB_DESTINATION}
+    /usr/local/${LIB_DESTINATION}
+    /usr/${LIB_DESTINATION}
+)
+

```

Comment:
What's the reason why we need an extra check for the debug version of xerces?

Reply:
if not, in debug mode the release library is used.

Reply:
For me the debug version of xerces is defined in UseLibPackCLbundler.cmake. Don't you use this file with the latest libpack?

Reply:
Thanks for the hint. I just made my changes to use this already defined set.

Reply:
Sorry but under Linux this modification won't change anything because XercesC_DEBUG_LIBRARIES is only set under Windows.

I will work on a modified FindXercesC.cmake version that follows that of CMake.